lib/chai/utils/inspect.js

```diff
@@ -230,6 +241,28 @@ function formatArray(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
+function formatTypedArray(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+
+  // Truncating TypedArray if it has more than 1000 elements
+  var l = value.length > 1000 ? 5 : value.length;
```

Comment:
This seems strange - if I had an array with 999 elements it would show all 999, but with 1000 it would only show 5?